CHANGELOG.md

```diff
@@ -6,6 +6,39 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Migration Notes
+
+From this release forward, Cumulus Core will be tested against PostgreSQL v13. Users
+should migrate their datastores to Aurora PostgreSQL 13.9+ compatible data
+stores as soon as possible after upgrading to this release.
+
+#### Engine Upgrade
+
+Users utilizing the `cumulus-rds-tf` module will have upgraded/had their
+database clusters forcibly upgraded at the next maintenance window after February 29, 2024.
+
+To upgrade your engine version, we recommend a manual (outside of
```

Comment:
So this is if you don't use the cumulus-rds-tf module? Or are we saying that all users should proceed with the manual upgrade to get ahead of AWS forcing the upgrade in Feb?

Reply:
This is for users utilizing `cumulus-rds-tf`.     Users *not* doing so are on their own with respect to how they update their cluster.